CIP-0006/README.md

```diff
@@ -79,7 +87,7 @@ The operator now:
 
 This re-registration of the main metadata file with the `extData.vkey` and the two URLs is only necessary once. Afterwards, the operator can update his extended metadata at any time, compute the new signature with the `cardano-cli stake-pool rawdata-sign` command, and publish both files at the existing `extDataUrl` and `extSigUrl`.
 
-### Extended Metadata structure
+## Rationale: how does this CIP achieve its goals?
 
 In the following we describe a first minimal version of the extended JSON file format
```

Comment:
```suggestion
In the following we describe a first minimal version of the extended JSON file format.
```
grammar :)

Reply:
@Ryun1 the period (rather than a colon) isn't a grammar mistake.  Mainly, it's a complete and grammatically correct sentence.  Also, the colon would normally be used if the "following" were following immediately, which it doesn't... instead it's distributed through this whole section & that's why I used the `.` rather than the `:`. 😛

Reply:
youre right!!